tools/d1-export-manifest.py

```
@@ -0,0 +1,290 @@
+    lines = []
+    for table in tables:
+        path = os.path.join(export_dir, table + ".json")
+        require_secure(path, table + ".json")
+        mtimes.append(os.stat(path).st_mtime)
+        lines.append(f"{sha256_file(path)}  {table}.json")
+
+    dump_mtime = os.stat(dump_path).st_mtime
+    dump_digest = sha256_file(dump_path)
+
+    # ── bounded consistency ──────────────────────────────────────────────
+    window = int(round(max(mtimes + [dump_mtime]) - min(mtimes + [dump_mtime])))
```

Ruff (0.16.2)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)